examples/01a_Si_diamond_pw_scf/INPUT

```diff
@@ -3,6 +3,7 @@ INPUT_PARAMETERS
 pseudo_dir		./	
 ntype			1	
 nbands			4
+#npool                   8
```

Comment:
This looks like a useless update

Reply:
It is indeed a useless modification, it has been fixed.

Reply:
> It is indeed a useless modification, it has been fixed.

@Asuna981002 You may 'Resolve conversation' by clicking the button below after patching.